WebAPI/Controllers/CategoriesController.cs

```diff
@@ -15,18 +15,32 @@ public CategoriesController(ICategoryService categoryService)
         _categoryService = categoryService;
     }
 
+    /// <summary>
+    /// Gets all Categories.
+    /// </summary>
+    /// <returns>List of Categories</returns>
     [HttpGet]
     public async Task<ActionResult<List<CategoryDetailDto>>> GetAll() =>
                 await _categoryService.GetAllAsync();
 
-    [HttpGet("{id}:guid")]
+    /// <summary>
+    /// Gets a specific Image.
+    /// </summary>
+    /// <param name="id"></param>
```

Comment:
I advise to always add description even for obvious params to make your docs consistent and clear. Do it, please, for the rest of your `<param>` tag